CONTRIBUTING.md

```diff
@@ -110,6 +110,17 @@ echo "Generate the release notes, and update the release page"
 echo "Inform various peoples"
 ```
 
+### Releasing chrome launcher.
+
+```sh
+cd chrome-launcher
+echo "build the launcher source code"
+yarn build
+echo "run npm version [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease | from-git]"
+echo "commit the change as a version bump"
```

Comment:
i prefer versioning with yarn because it's interactive, but w/e. also in both cases the commit (and tag) is made automatically.

```
yarn version
echo "version bump will be committed"
```